cli/command/container/utils.go

```diff
@@ -38,6 +38,7 @@ func waitExitOrRemoved(ctx context.Context, apiClient client.APIClient, containe
 	go func() {
 		select {
```

Comment:
nit: 
Maybe we could close in a defer in the goroutine scope so that it's less likely to be missed in future?

```suggestion
	go func() {
	    defer close(statusC)
		select {
```